docs/source/contributing/setup.rst

```diff
@@ -0,0 +1,178 @@
+.. _contributing/setup:
+
+================================
+Setting up a development install
+================================
+
+System requirements
+===================
+
+JupyterHub can only run on MacOS or Linux operating systems. If you are
+using Windows, we recommend using `VirtualBox <https://virtualbox.org>`_ 
```

Comment:
would JupyterHub work on windows subsystem for linux? If so that'd be much simpler than virtualbox...maybe I can try this out